.gitattributes

```diff
@@ -0,0 +1,12 @@
+poetry.lock binary
+yarn.lock binary
```

Comment:
In my experience resolving issues with yarn, the lockfile is sometimes useful to look at (either in whole or in diffs). My reccommendation/preference for `yarn.lock` (and probably the others) would be to use
```gitattributes
yarn.lock merge=binary linguist-generated
```
By using `merge=binary`, git shows a normal textual diff, but `git merge` shows something like
```
warning: Cannot merge binary files: a.txt (HEAD vs. c)
Auto-merging a.txt
CONFLICT (content): Merge conflict in a.txt
Automatic merge failed; fix conflicts and then commit the result.
```
And `linguist-generated` is what github recommends (https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github) for auto-generated files; it causes github to collapse the diff by default.